guides/source/active_storage_overview.md

```diff
@@ -858,15 +858,16 @@ transformations depending on the image's format:
 
 2. If `quality` is not specified, the variant processor's default quality for the format will be used.
 
-The default processor for Active Storage is MiniMagick, but you can also use
-[Vips][]. To switch to Vips, add the following to `config/application.rb`:
+The default processor for Active Storage is now [Vips][], but if using
+configuration defaults below 7.0 it will be MiniMagick. To switch between the
+two, add the following to `config/application.rb`:
 
 ```ruby
-config.active_storage.variant_processor = :vips
+config.active_storage.variant_processor = :vips # or :mini_magick
```

Comment:
```suggestion
config.active_storage.variant_processor = :mini_magick # default :vips
```

Reply:
If using my suggestion above (that links to the configuration guide), you can remove this snippet.